protopipe_grid_interface/scripts/submit_jobs.py

```diff
@@ -607,8 +607,13 @@ def main():
 
         # this sends the job to the GRID and uploads all the
         # files into the input sandbox in the process
+
+        job = dirac.submitJob(j)
+        if not job["OK"]:
+            raise RuntimeError(f"Job submission failed: {job}")
```

Comment:
yes, that would be better.  I also wondered if it really should stop (raise an exception) if one job fails, or if it should just log a warning and continue? Is it easy to re-start jobs that failed later? I guess it will not re-run an existing job?

Reply:
remember that this is the submission of the job, not the job itself: usually, if a job fails to be submitted it's due to a common issue (to all planned submissions)

one can certainly skip the job in case it's e.g. a network issue just at that moment, but in that case the solution should be a while loop catching a network specific exception I guess

Currently a job won't be submitted either if it's already running, or if the output file is already present at the specified output directory on the file catalog